static/skywire-visor.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Skywire Visor
+After=network.target
+
+[Service]
+Type=simple
+User=root
+Group=root
+ExecStart=/usr/bin/skywire-visor /etc/skywire-visor.json
```

Comment:
This will be changed to ` ExecStart=/usr/bin/skywire-visor -c /etc/skywire-visor.json` after the latest changes to `develop`.

Reply:
noted

Reply:
fixed